components/inputs/input-date.js

```diff
@@ -161,12 +165,14 @@ class InputDate extends LocalizeCoreElement(LitElement) {
 			</div>
 			<d2l-dropdown ?disabled="${this.disabled}" no-auto-open>
 				<d2l-input-text
+					aria-invalid="${this._invalidInput ? 'true' : 'false'}"
```

Comment:
Calling `validate` automatically adds `aria-required` to your component so (with the current pattern) it'd be preferable to add an `aria-required` property and pass that down to `d2l-input-text`. That way you don't need to worry about tracking whether its currently valid or invalid.